symengine/tests/basic/test_polynomial.cpp

```diff
@@ -595,3 +601,437 @@ TEST_CASE("UnivariatePolynomial expand", "[UnivariatePolynomial][expand]")
                             "a**2)*x**7 + (1 + 6*a)*x**6 + (3 + 3*a)*x**5 + "
                             "3*x**4 + x**3");
 }
+
+TEST_CASE("Constructing MultivariateIntPolynomial using from_dict",
+          "[MultivariateIntPolynomial]")
+{
```

Comment:
Please remove the changes in test_polynomial.cpp, now that the tests have been added in the other file.